kolibri/utils/lru_cache.py

```diff
@@ -0,0 +1,185 @@
+try:  # NOQA
```

Comment:
It's optimized now by moving it to `get_version`, any call to this function will be cached in the running python instance.